cunit/unit-timeofday.c

```diff
@@ -234,124 +248,152 @@ static int real_gettimeofday(struct timeval *tv, ...)
 #endif
 }
 
-/* provide a function to hide the libc weak alias */
-EXPORTED int gettimeofday(struct timeval *tv, ...)
+EXPORTED time_t real_time(time_t *tp)
 {
-    to_timeval(transform(now()), tv);
-    return 0;
+    /* XXX can't find the name of the real function to wrap, so mimic it... */
```

Comment:
I don't understand this comment, can you explain?

Also why is it a TODO?

Reply:
What I actually want to do here is call through to the real glibc function, like what `real_gettimeofday()` and such do, with the big mess of ifdefs to choose the correct internal name.

But for this one and one other (iirc), I couldn't find the right name to invoke the real function after a reasonable amount of exploration, so I gave up and just implemented something equivalent instead.